File: AC-ABC-TDD/177/a.py
def is_intime(d, t, s: int) -> bool:
    if s*t >= d:
        return True
    else:
        return False


d, t, s = map(int, input().split())
if is_intime(d, t, s):
    print("Yes")
else:
    print("No")

File: AC-ABC-TDD/177/a.go
package main

import "fmt"

func main() {
	// s*t >= d ならok
	// O(1)
	var d, t, s int
	fmt.Scan(&d, &t, &s)
	if s*t >= d {
		fmt.Println("Yes")
	} else {
		fmt.Println("No")
	}
}

// package a
//
// func IsInTime(d, t, s int) bool {
// 	if s*t >= d {
// 		return true
// 	} else {
// 		return false
// 	}
// }
